.env.sample

```diff
@@ -0,0 +1,4 @@
+REACT_APP_CLIENT_ID=""
+REACT_APP_OKTA_ISSUER_URI=""
+REACT_APP_BACKENDURL=""
+REACT_APP_MAPBOX_TOKEN=""
```

Comment:
You may want to add one for REACT_APP_MAPQUEST_API_KEY because that's a new variable that will be required from now on in frontend .env